### ISSUE TITLE
V110 won't upload logs

### ISSUE THREAD

**Issue by lomonossof** [mendhak/gpslogger#883]
Hi!

I have a problem with version 110: Everytime I try to upload a tracklog to my NextCloud webspace I get a AuthPolicy error, despite username and password are correct.
After a downgrade to V109 it all works well again 

Thomas

**Comment by mendhak** [comment]
AuthPolicy, is that an error on the NextCloud side? Can you post the full error/stack trace, if it's still in your logs.  There might be some clue in the full trace, because I don't know what that error means right now.

**Comment by mendhak** [comment]
I got it.  I just tried the NextCloud docker container, and tried an upload, and I see the problem, it's complaining about a missing class.  Proguard stripped out the wrong class.  I found the answer here:  https://github.com/nextcloud/android-library/issues/336#issuecomment-531694658

**Comment by lomonossof** [comment]
Wow! You're fast! I just _finished thinking about_ upgrading again to make a screenshot of the error message. Seems I'm gettin' old ... ;-)

**Comment by mendhak** [comment]
Haha thanks, I did have to act quickly since I felt a bit guilty - I tested as much as I could but I must have missed this. 

So I've now tagged this latest change.  F-Droid will pick it up automatically and build it.  But it will take a few days to appear on their store.  Once that arrives it should work.  I did test with a Docker container and it worked so that should be good.  

Side note - it's kind of nice that in F-Droid is you were able to pick a version to roll back to.  I don't recall being able to do that with the Play Store.
